.github/CODEOWNERS

```diff
@@ -13,3 +13,4 @@
 /charts/pentaho-carte/ @ejcsid
 /charts/wjh-rechner/ @maximilian-zollbrecht
 /charts/zammad-ldap-sync/ @klml
+/charts/refarch-gateway/ @it-at-m/refarch-maintainer
```

Comment:
@eidottermihi would it be ok to use a group as codeowner?
If yes the group needs to get permissions on this repository that the codeowner file is valid.

Reply:
fine by me, added [refarch-maintainer](https://github.com/orgs/it-at-m/teams/refarch-maintainer) team